frontend/src/components/transfers/TransferDetails.js

```diff
@@ -0,0 +1,177 @@
+import React, { useEffect, useState } from 'react'
+import { connect } from 'react-redux'
+import * as actions from '../../store/actions'
+import ThingahaName from '../common/ThingahaName'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+import Paper from '@material-ui/core/Paper'
+import EventRoundedIcon from '@material-ui/icons/EventRounded'
+import MonetizationOnIcon from '@material-ui/icons/MonetizationOn'
+import ReplyRoundedIcon from '@material-ui/icons/ReplyRounded'
+import ReceiptIcon from '@material-ui/icons/Receipt'
+import EditIcon from '@material-ui/icons/EditRounded'
+import TransferForm from './TransferForm'
+import ExtraFundCurrentAmount from '../extraFunds/ExtraFundCurrentAmount'
+
+const Wrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 20px;
+`
+
+const TopIconContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 1rem 1rem;
+  justify-content: space-between;
+
+  & .edit {
+    cursor: pointer;
+  }
+`
+
+const TransferDetailWrapper = styled(Paper)`
+  display: flex;
+  /* justify-content: flex-start; */
+  flex-direction: row;
+  align-items: flex-start;
+  /* margin-top:2rem; */
+  padding: 1rem 1rem;
+  justify-content: space-between;
+
+  & .infoText {
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  & .iconTextWrapper {
+    display: inline-flex;
+    flex-direction: row;
+    padding: 8px 4px;
+  }
+
+  & .smallText {
+    padding-left: 0.5rem;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    align-self: flex-end;
+  }
+
+  & .smallTextLabel {
+    min-width: 5.5rem;
+    color: ${(props) => props.theme.palette.text.tertiary};
+    padding-left: 0.5rem;
+    font-size: 1rem;
+    line-height: 1.25rem;
+    align-self: flex-end;
+  }
+`
+
+const labelTransferTitle = 'Transfer Record'
+const labelExtraFunds = 'Current Extra Fund'
```

Comment:
Can you please switch to "Extra Funds from this Transfer" ?

Reply:
Oops, I only show the current Extra Funds, So, I need to change code for each transactions.

And how about a transition that doesn’t register in Extra Funds?

Reply:
@sisminnmaw  Good point. Let's only show Extra Fund section only if there is any extra fund registered for that transfer.